server/api/controllers/users/delete-user.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  friendlyName: "Delete users",
+
+  description: "",
+
+  inputs: {
+    userId: {
+      required: true,
+      type: "string",
+    },
+  },
+
+  exits: {
+    success: {
+      description: "User account deleted successfully",
+    },
+    invalid: {
+      description: "Failed to delete user account",
+    },
+
+    missingRequiredFields: {
+      statusCode: 409,
+      description: "Please fill in the required fields.",
+    },
+  },
+
+  fn: async function ({ userId }, exits) {
+    if (userId) {
```

Comment:
this is much cleaner esp since we also do not need to throw "missingRequiredFields" since sails already handles that when you defined the input to be `required=true`